src/app/zap-templates/call-command-handler-src.zapt

```diff
@@ -48,7 +48,7 @@ EmberAfStatus emberAfClusterSpecificCommandParse(EmberAfClusterCommand * cmd)
         switch (cmd->apsFrame->clusterId)
         {
         {{#all_user_clusters}}
-        {{#if (isEnabled enabled)}}
+        {{#if (user_cluster_has_enabled_command name side)}}
         {{#if (isClient side) }}
         case ZCL_{{asDelimitedMacro define}}_ID :
             result = emberAf{{asCamelCased name false}}Cluster{{asCamelCased side false}}CommandParse(cmd);
```

Comment:
The previous code uses to execute the following line when the there is no cluster commands associated with the cluster itself:
```
result = status(false, true, cmd->mfgSpecific);
```

Should we add it back ?

Reply:
Good idea. This way we'll be able to differentiate between an Unsupported cluster and an enabled cluster with no commands.